packages/mdl-drawer/temporary/index.js

```diff
@@ -57,6 +57,8 @@ export default class MDLTemporaryDrawer extends MDLComponent {
           this.drawer.removeEventListener(util.remapEvent(evt), handler),
       registerTransitionEndHandler: handler => this.drawer.addEventListener('transitionend', handler),
       deregisterTransitionEndHandler: handler => this.drawer.removeEventListener('transitionend', handler),
+      registerDocumentKeydownHandler: handler => document.addEventListener('keydown', handler),
```

Comment:
That would mean we would have to focus-trap within the drawer. I feel like we should probably be doing this anyway, but we currently don't. For example, if you tab-focus on the icon, and then hit enter to activate the menu, you're still focused on the menu icon, meaning that would receive the `keyup` event. Thoughts?